doc/snippets/MagnumPluginsBasisImporter.cpp

```diff
@@ -0,0 +1,44 @@
+#include <Corrade/PluginManager/Manager.h>
+#include <Corrade/Utility/ConfigurationGroup.h>
+
+#include <Magnum/Trade/AbstractImporter.h>
+
+#ifdef __has_include
+#if __has_include(<MagnumPlugins/BasisImporter/importStaticPlugin.cpp>)
+/* [static-import] */
+/* No need to do this if you use CMake */
+#include <MagnumPlugins/BasisImporter/importStaticPlugin.cpp>
+/* [static-import] */
+#endif
+#endif
```

Comment:
No matter how hard I try, I can't find this snippet being used anywhere. Since it's the same for all plugins I don't think it makes sense to list it for every plugin, so ... remove? :)